eagle-core/eagle-policy/eagle-policy-base/src/main/java/org/apache/eagle/alert/entity/AlertAPIEntity.java

```diff
@@ -78,19 +81,22 @@ public void setRemediationCallback(String remediationCallback) {
 		_pcs.firePropertyChange("remediationCallback", null, null);
 	}
 
+    @JsonIgnore
 	public String getAlertContext() {
 		return alertContext;
 	}
 
+    @JsonProperty("alertContext")
```

Comment:
Avoid to use JsonProperty annotation like this which makes the code too complex to read.